builders/go/README.md

```diff
@@ -22,6 +23,112 @@ This document explains how to use the builder for Golang projects.
 
 To generate provenance for a golang binary, follow the steps below:
 
+### Configuration file
+
+Define a configuration file called `.slsa-goreleaser.yml` in the root of your project.
+
+```yml
+# Version for this file.
+version: 1
+
+# (Optional) List of env variables used during compilation.
+env:
+  - GO111MODULE=on
+  - CGO_ENABLED=0
+
+# (Optional) Flags for the compiler.
+flags:
+  - -trimpath
+  - -tags=netgo
+
+# The OS to compile for. `GOOS` env variable will be set to this value.
+goos: linux 
+
+# The architecture to compile for. `GOARCH` env variable will be set to this value.
+goarch: amd64
+
+# (Optional) Entrypoint to compile. (Optional)
+# main: ./path/to/main.go
+
+# (Optional) Working directory. (default: root of the project)
+# dir: /path/to/dir
+
+# Binary output name.
+# {{ .Os }} will be replaced by goos field in the config file.
+# {{ .Arch }} will be replaced by goarch field in the config file.
+binary: binary-{{ .Os }}-{{ .Arch }}
+
+# (Optional) ldflags generated dynamically in the workflow, and set as the `env` input variables in the workflow.
+ldflags:
+  - "{{ .Env.VERSION_LDFLAGS }}"
+```
+
+### Migration from goreleaser
+
+If you are already using Goreleaser, you may be able to migrate to our builder using multiple config files for each build. However, this is cumbersome and we are working on supporting multiple builds in single config file for future releases. 
```

Comment:
This seems a bit confusing to me. As a goreleaser user I'm not exactly sure what I should do. Is it worth mentioning that `.slsa-goreleaser.yml` supports a subset of goreleaser's configuration file?

Reply:
I've updated. PTAL. Do you think I should provide an example?

Reply:
I think this is ok for now. Thanks.